src/css/base/_variables.scss

```diff
@@ -9,7 +9,7 @@ $coa-font-serif: $coa-font-sans;
 $coa-font-regular: 400;
 $coa-font-semibold: 600;
 $coa-font-bold: 700;
-$coa-font-extrabold: 800;
+$coa-font-extrabold: 700;
```

Comment:
we should talk about this...I don't like having the same font-weight being in two variables...can we audit where these variables are used? and/or continue to have a font weight of 800 available to use (maybe this isn't possible with IBM Plex Sans, though)

Reply:
IBM Plex Sans maxes out at boldness/font-weight 700. I can audit and replace instances of $coa-font-extrabold with $coa-font-bold.

Reply:
thumb up emoji